sidebars.js

```diff
@@ -326,7 +326,7 @@ const sidebars = {
             },
             {
               type: 'category',
-              label: 'Manage Clusters',
+              label: 'Cluster Administration',
```

Comment:
This is another special case. The page is titled "Cluster Administration" and isn't a how-to. Instead, it contains a table of actions you can perform on clusters